schemas/rum/_vital-info-schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "rum/_vital-info-schema.json",
+  "title": "RumVitalInfo",
+  "type": "object",
+  "description": "Schema of properties for a vital",
+  "required": ["view_min", "view_max", "view_average"],
+  "properties": {
+    "view_min": {
+      "type": "number",
+      "description": "The minimum value seen for this vital in the view.",
+      "readOnly": true
+    },
+    "view_max": {
+      "type": "number",
+      "description": "The maximum value seen for this vital in the view.",
+      "readOnly": true
+    },
+    "view_average": {
+      "type": "number",
+      "description": "The minimum value seen for this vital in the view.",
+      "readOnly": true
+    },
+    "vital_max": {
```

Comment:
AFAIK, `vital_max` is not related to the view? It seems weird to include it in the View event.

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏remove this value. Users can use the datadog UI to see the max value for all views from a session.

Reply:
This was meant as "the maximum possible value" not the maximum we've seen. The use case generally is FPS on a mobile device, which can have a different max depending on the device. An app running at 59 FPS when the max is 60 is a very different metric from an app running at 59 FPS when the max is 120+, if that makes sense.

Reply:
I see! Since it is related to the device, I would suggest to put it in the device field, ex: `@device.max_fps`

Reply:
So, one idea for putting all this info together was to make creating UI for these metrics easier. The idea being that if we can come up with a generalized way of showing a "vital" (for lack of a better term) then all you need to know is the name of the property in the "vitals" object.